### ISSUE TITLE
Add support of card events: openUrl, showImage, postBack

### ISSUE THREAD

**Issue by poul-kg** [Microsoft/BotFramework-Emulator#149]
```JavaScript
// Function to create ThumbnailCard from given location object
function getLocationCard(session, location) {
    const card = new builder.ThumbnailCard(session);
    card.title(location.Name);
    card.subtitle(location.Address1);
    if (location.Phone) {
        card.text(location.Phone);
    }
    card.images([builder.CardImage.create(session, location.logo_url)]);
    // ISSUE HERE
    card.tap(new builder.CardAction.openUrl(session, 'https://google.com'));
    return card;
}

const cards = locations.map(function (location) {
        return getLocationCard(session, location)
});
const message = new builder.Message(session).attachments(cards).attachmentLayout('list');
session.send(message);
```

I'm using code above to create thumbnail cards and attach onTap events.

But when I use emulator and click any item, nothing happens.

**Comment by eanders-MS** [comment]
Hi @poul-kg, thanks for taking the time to report this. This issue has been fixed in the codebase and will be available in the next release.

**Comment by eanders-MS** [comment]
Fixed with build 3.4.26
